data/data/agent/files/usr/local/bin/start-agent.sh.template

```diff
@@ -2,14 +2,5 @@
 
 source common.sh
 
->&2 echo "Waiting for infra-env-id to be available"
-INFRA_ENV_ID=""
-until [[ $INFRA_ENV_ID != "" && $INFRA_ENV_ID != "null" ]]; do
-    sleep 5
-    >&2 echo "Querying assisted-service for infra-env-id..."
-    INFRA_ENV_ID=$(curl -s -S '{{.ServiceBaseURL}}/api/assisted-install/v2/infra-envs' | jq -r .[0].id)
-done
-echo "Fetched infra-env-id and found: $INFRA_ENV_ID"
-
```

Comment:
Maybe this script isn't required anymore, and the exec command moved directly in the ExecStart of the agent.service.template file?

Reply:
Done